packages/block-library/src/post-content/inner-blocks.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityBlockEditor } from '@wordpress/core-data';
+import { InnerBlocks } from '@wordpress/block-editor';
+
+/**
+ * Renders the content of the current post in a controlled inner block area.
+ *
+ * @param {Object} Props Component props.
+ * @param {string} Props.postType The postType to use for the entity blocks.
+ * @return {Function} Controlled InnerBlocks of the current post.
```

Comment:
The alignment is off. We also tend not to use JSDoc for internal components when they're self-explanatory.

Reply:
JSDoc is still useful for typechecking, though in this case the types aren't really that helpful. (The return type should specify that it's returning a component, rather than using the generic `Function` type.)